.github/workflows/chatops-command-dispatch.yml

```diff
@@ -89,6 +89,8 @@ jobs:
           issue-type: pull-request
           commands: |
             ci-problems
+            disable-cache
+            enable-cache
```

Comment:
🔧 I'd prefix with `ci-` to make scope of the commands clear (sorry I didn't catch this earlier in your proposal).